apps/condo/lang/pt/messages/BILLING_RECEIPT_FILE_ADDED/email.html.njk

```diff
@@ -0,0 +1,24 @@
+<html style="fmargin: 0; font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif; font-size: 18px; font-weight: 400; line-height: 32px; text-align: left;">
+<body style="margin: 0; font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif; font-size: 18px; font-weight: 400; line-height: 32px; text-align: left;">
+<p style="font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif; font-size: 18px; font-weight: 400; line-height: 32px; text-align: left;">
+    Hello!<br>
+    New billing receipt available.<br>
+    <br>
+    To download and pay for a receipt please use Unitify application.<br>
+    <br>
+    Install application:<br>
+</p>
+<ul style="font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif; font-size: 18px; font-weight: 400; line-height: 32px; text-align: left;">
+    <li> In App Store, search for "Domiki" application</li>
+    <li> In Google Play, search for "Unitify" application</li>
+    <li> Scan QR-code on receipt or open <a href="https://clck.ru/376TNL">link</a></li>
+</ul>
+<p style="font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif; font-size: 18px; font-weight: 400; line-height: 32px; text-align: left;">
+    Add account number in application. In «Payment» section choose receipt, click «Additional» - «Open receipt». You can download receipt at this section.
```

Comment:
Better to add ability to store company name and mobile application names in variable.. not sure I like hardcoding company names ..